src/lib/alarmMgr/AlarmManager.h

```diff
@@ -80,6 +84,9 @@ class AlarmManager
   bool notificationError(const std::string& url, const std::string& details);
   bool notificationErrorReset(const std::string& url);
 
+  bool notificationQueuesResets(const std::string& details);
+  bool notificationQueue(const std::string& url, const std::string& details);
```

Comment:
```suggestion
  bool notificationQueuesResets(const std::string& service);
  bool notificationQueue(const std::string& service, const std::string& details);
```

Let's use proper variable names ;)

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/4375/commits/2e046041b810bc0c7791e35bafdf154ac2118df5